client/js/state.js

```diff
@@ -9,24 +9,116 @@ export default {
   people: [{
     _id: 'c5c84f5a-bc9c-11e5-9912-ba0be0483c18',
     name: 'Scott Grogan'
+  }, {
+    _id: '7ca8de1f-e565-4b7f-9914-7284ea582933',
+    name: 'David Rieman'
+  }, {
+    _id: '7cf56c45-2712-4f59-a3e5-64d174614534',
+    name: 'Ben McEldowney'
+  }, {
+    _id: 'bd0cb484-60e0-474b-a7f0-dbf91857c9e9',
+    name: 'Elrond Sheppard'
+  }, {
+    _id: 'cc1c2592-7aab-4bc6-a513-108e29971b43',
+    name: 'Chris Johnson'
+  }, {
+    _id: '7fd8cf79-133e-433c-a373-e78a63fb1d18',
+    name: 'Daniel Harrie'
   }],
 
   // High-level goals with descriptions and deadlines
   goals: [{
-    _id: 'db184996-bc9c-11e5-9912-ba0be0483c18',
-    _parent: 'c5c84f5a-bc9c-11e5-9912-ba0be0483c18',
-    name: 'Lead the XP reading group',
-    description: 'Descended from astronomers bits of moving fluff, paroxysm of global death, Vangelis. Billions upon billions another world tesseract a still more glorious dawn awaits rings of Uranus something incredible is waiting to be known across the centuries cosmic ocean the carbon in our apple pies network of wormholes, white dwarf worldlets colonies, muse about.',
-    created_on: '2016-01-01T15:55:45-08:00',
-    updated_on: '2016-01-07T15:55:45-08:00'
+    _id: "4c658e60-f39f-427c-a5cd-95129b60e929",
+    _parent: "7fd8cf79-133e-433c-a373-e78a63fb1d18",
+    name: "Collaboratively administrate empowered markets via plug-and-play networks",
+    description: "Dynamically procrastinate B2C users after installed base benefits. Dramatically visualize customer directed convergence without revolutionary ROI.",
```

Comment:
Yep.  Not the best generator ever since it always generates the same thing in the same order, but adding paragraphs get’s something new and it’s hard to beat in the context of a scorecard app.

From: Scott Grogan [mailto:notifications@github.com]
Sent: Thursday, March 24, 2016 8:15 AM
To: ninjascribble/people <people@noreply.github.com>
Cc: Daniel Harrie <dharrie@weareratio.com>
Subject: Re: [people] Created My Team page (#1)


In client/js/state.js<https://github.com/ninjascribble/people/pull/1#discussion_r57332477>:

>    }],

>

>    // High-level goals with descriptions and deadlines

>    goals: [{

> -    _id: 'db184996-bc9c-11e5-9912-ba0be0483c18',

> -    _parent: 'c5c84f5a-bc9c-11e5-9912-ba0be0483c18',

> -    name: 'Lead the XP reading group',

> -    description: 'Descended from astronomers bits of moving fluff, paroxysm of global death, Vangelis. Billions upon billions another world tesseract a still more glorious dawn awaits rings of Uranus something incredible is waiting to be known across the centuries cosmic ocean the carbon in our apple pies network of wormholes, white dwarf worldlets colonies, muse about.',

> -    created_on: '2016-01-01T15:55:45-08:00',

> -    updated_on: '2016-01-07T15:55:45-08:00'

> +    _id: "4c658e60-f39f-427c-a5cd-95129b60e929",

> +    _parent: "7fd8cf79-133e-433c-a373-e78a63fb1d18",

> +    name: "Collaboratively administrate empowered markets via plug-and-play networks",

> +    description: "Dynamically procrastinate B2C users after installed base benefits. Dramatically visualize customer directed convergence without revolutionary ROI.",

These are fantastic descriptions! http://www.cipsum.com/? [:stuck_out_tongue_closed_eyes:]

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ninjascribble/people/pull/1/files/c1fcac520bf1636c5db4814587ee9de8df727f2b#r57332477>